sensor/common/sensor/sensor.go

```diff
@@ -214,6 +214,12 @@ func (s *Sensor) Start() {
 			utils.Should(errors.Wrap(err, "Failed to create scanner definition route"))
 		}
 		customRoutes = append(customRoutes, *route)
+
+		repoMappingRoute, err := s.newScannerRepoMappingRoute(s.centralEndpoint)
+		if err != nil {
+			utils.Should(errors.Wrap(err, "Failed to create scanner v4 repository mapping data route"))
+		}
+		customRoutes = append(customRoutes, *repoMappingRoute)
```

Comment:
Should we run both V2 and V4 in the Secured Cluster? Perhaps we should disable one endpoint or route to different Central endpoints based on Scanner V4/V2 enablement status? Need to clarify this.

Reply:
I decide to leave the sensor part to another PR, what do you think?

Reply:
Moving the sensor to another PR is OK. Track it somewhere for planning.